public/js/graph/react_graph.js

```diff
@@ -579,8 +595,7 @@ var Node = React.createClass({
             this.setState({status: newState}, function () {
                 setCookie(nodeId, newState);
                 this.props.childs.forEach(function (node) {
-                    var currentNode = svg.refs['nodes'].refs[node] ||
-                                      svg.refs['bools'].refs[node];
+                    var currentNode = refLookUp(node, this.props.svg);
```

Comment:
@klmerryn I should have caught this sooner, but this doesn't work. Use of `this` inside an anonymous function no longer refers to the "outer this" (in this case, the Node instance). Note that the original code did not refer to `this` at all.

Also, ask Christine/Ryan about this problem, as they've come across it before.

Note: this happens in multiple places.